modules/Documents/tpls/detailHeader.tpl

```diff
@@ -0,0 +1,46 @@
+{*
+/*********************************************************************************
+ * SugarCRM Community Edition is a customer relationship management program developed by
+ * SugarCRM, Inc. Copyright (C) 2004-2013 SugarCRM Inc.
+
+ * SuiteCRM is an extension to SugarCRM Community Edition developed by Salesagility Ltd.
+ * Copyright (C) 2011 - 2014 Salesagility Ltd.
```

Comment:
@jsamelko Can you update with appropriate header :+1: 
https://docs.suitecrm.com/community/contributing-code/coding-standards/

Reply:
@samus-aran No problems - updated. Thanks